templates/arista_eos_show_mac_address-table.textfsm

```diff
@@ -1,10 +1,35 @@
-Value MAC_ADDRESS (\w+?\.\w+?\.\w+?|\w+?\:\w+?\:\w+?\:\w+?\:\w+?\:\w+?)
-Value TYPE (\w+?)
-Value VLAN (\d+?)
-Value DESTINATION_PORT (\w+\s\w+\s\w+\s\w+|\w+\s\w+\s\w+|\w+\s\w+|\w+?)
-Value MOVES (\d+?)
+Value MAC_ADDRESS (\w+\.\w+\.\w+|\w+\:\w+\:\w+\:\w+\:\w+\:\w+)
+Value TYPE (\w+)
+Value VLAN (\d+)
+Value List DESTINATION_PORT ((\w+\/\w+)|\w+)
+Value MOVES (\d+)
 Value LAST_MOVE (.+)
 
 Start
-  ^\s*${VLAN}\s+?${MAC_ADDRESS}\s+?${TYPE}\s+?${DESTINATION_PORT}\s+?${MOVES}\s+${LAST_MOVE}\sago$$ -> Record
-  ^\s*${VLAN}\s+?${MAC_ADDRESS}\s+?${TYPE}\s+?${DESTINATION_PORT}$$ -> Record
\ No newline at end of file
+  ^\s*Vlan\s+(M|m)ac\sAddress\s+Type\s+Ports\s+Moves\s+Last\sMove\s*$$ -> MAT
+  ^\s*Vlan\s+(M|m)ac\sAddress\s+Type\s+Ports\s*$$ -> MMAT
+  ^\s*$$
+  ^\s*(M|m)ac\sAddress\sTable\s*$$
+  ^\s*Multicast\s(M|m)ac\sAddress\sTable\s*$$
+  ^-*$$
+  # ^(-*\s*)*$$
+  ^. -> Error
+
+MAT
+  # capture the mac address table
+  ^\s*${VLAN}\s+${MAC_ADDRESS}\s+${TYPE}\s+${DESTINATION_PORT}\s*$$ -> Record
+  ^\s*${VLAN}\s+${MAC_ADDRESS}\s+${TYPE}\s+${DESTINATION_PORT}\s+${MOVES}\s+${LAST_MOVE}\sago$$ -> Record
+  ^\s*$$ -> Start
+
+MMAT
+  # capture the multicast mac address table
+  # key on the record
+  ^\s*\d+\s+(\w+\.\w+\.\w+|\w+\:\w+\:\w+\:\w+\:\w+\:\w+)\s+\w+\s*.*$$ -> Continue.Record
+  # match the record with one to five destination ports more.
+  ^\s*${VLAN}\s+${MAC_ADDRESS}\s+${TYPE}\s+${DESTINATION_PORT}$$ -> Record
+  ^\s*${VLAN}\s+${MAC_ADDRESS}\s+${TYPE}\s+${DESTINATION_PORT}\s -> Continue
+  ^\s*\d+\s+(\w+\.\w+\.\w+|\w+\:\w+\:\w+\:\w+\:\w+\:\w+)\s+\w+\s+((\w+\/\w+|\w+)\s){1}${DESTINATION_PORT}\s* -> Continue
+  ^\s*\d+\s+(\w+\.\w+\.\w+|\w+\:\w+\:\w+\:\w+\:\w+\:\w+)\s+\w+\s+((\w+\/\w+|\w+)\s){2}${DESTINATION_PORT}\s* -> Continue
+  ^\s*\d+\s+(\w+\.\w+\.\w+|\w+\:\w+\:\w+\:\w+\:\w+\:\w+)\s+\w+\s+((\w+\/\w+|\w+)\s){3}${DESTINATION_PORT}\s* -> Continue
+  ^\s*\d+\s+(\w+\.\w+\.\w+|\w+\:\w+\:\w+\:\w+\:\w+\:\w+)\s+\w+\s+((\w+\/\w+|\w+)\s){4}${DESTINATION_PORT}$$ -> Continue
+  ^\s*$$ -> Start
```

Comment:
Do the same here, transition back on the `Total mac` and then add the required two lines in this state as well.

```suggestion
  ^Total\s+mac -> Start
  ^\s*$$
  ^. -> Error
```